core/src/main/java/com/scalar/db/common/error/CoreError.java

```diff
@@ -115,10 +115,10 @@ public enum CoreError implements ScalarDbError {
       "The clustering key is not properly specified. Operation: %s",
       "",
       ""),
-  NOT_SUPPORTED_IN_COMMUNITY_EDITION(
+  AUTH_NOT_ENABLED(
       Category.USER_ERROR,
       "0022",
-      "This feature is not supported in the ScalarDB Community edition",
+      "ScalarDB Auth is not enabled. Enable ScalarDB Auth to use this feature. Note that this feature is supported only in the ScalarDB Enterprise edition",
```

Comment:
I was a little confused at first since I thought `ScalarDB Auth` and `this feature` were two different things. How about we revise this as follows?:

```suggestion
      "ScalarDB Auth is not enabled. To use ScalarDB Auth, you must enable it. Note that this feature is supported only in the ScalarDB Enterprise edition",
```